explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -131,8 +131,8 @@ private object SpectroscopyModesTable extends TableHooks:
     entry:  SpectroscopyModeRow,
     result: EitherNec[ItcQueryProblems, ItcResult]
   ):
-    lazy val exposureTime: Option[Int] =
-      result.toOption.collect { case ItcResult.Result(e, t) => e.toMillis.toInt * t }
+    lazy val totalItcTime: Option[Double] =
```

Comment:
Is there a reason to use Double vs Long? I think `e.toMillis` is a Long, right?

Reply:
It is but the number is very large and it produces wrapping, I was a bit surprised too